src/app/shared/components/pagenotfound/pagenotfound.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core'
+
+@Component({
+  selector: 'cav-pagenotfound',
+  templateUrl: './pagenotfound.component.html',
+  styleUrls: ['./pagenotfound.component.scss'],
+})
+export class PagenotfoundComponent implements OnInit {
```

Comment:
Please, rename the component using Pascal Case notation. Thanks.

Reply:
Sorry, done!

Reply:
Thanks mate :)